### DISCUSSION TITLE
Prices of GitHub-hosted runners for enterprise plan

### DISCUSSION THREAD

**Discussion by MarounMaroun** [community/community#60580] [Enterprise]
### Select Topic Area

Question

### Body

Hi,

We're currently on the GitHub enterprise plan, where we have 50,000 minutes included. I'd like to have a new runner group, with larger machines.

How the billing works in this case? Will the minutes be still available for us?

**Comment by Akash1134** [comment]
Hi @MarounMaroun ,

Please use our [pricing calculator](https://github.com/pricing/calculator) and [pricing FAQ](https://github.com/pricing) to get the best suited plan for your needs

For billing related details, see: [About billing for GitHub Actions](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions)

Thanks!

**Reply by MarounMaroun** [reply]
@Akash1134 Thanks for your reply, however, neither of the links answer my question.
